packages/api/src/command/medical/patient/normalize-patient.ts

```diff
@@ -0,0 +1,145 @@
+import { PatientData, splitName } from "../../../domain/medical/patient";
+
+// Define default values for each field
+const defaultValues = {
```

Comment:
if "default values" are inputted, we should either remove them or reject the patient altogether. Having these values in our system will pollute downstream patient matching. And people will put these sort of things in without thinking sometimes just to fill in fields (even if those fields aren't nessecarily required. This list isn't exhaustive by any means but conceptually, removing default values is an important part of all the different MPI systems I researched